docs/source/user/installing.rst

```diff
@@ -185,18 +185,28 @@ vary with target operating system and hardware. The following lists them all
   * ``numpy``
   * ``llvmlite``
   * Compiler toolchain mentioned above
-  * OpenMP C headers and runtime libraries compatible with the compiler
-    toolchain mentioned above and accessible to the compiler via standard flags
-    (Linux, Windows).
 
-* Optional build time:
-
-  * ``llvm-openmp`` (OSX) - provides headers for compiling OpenMP support into
+* Build time environment variables:
+  * ``NUMBA_NO_OPENMP``
+    Set this environment variable to ``1`` when building
```

Comment:
CI is complaining about unexpected indentation on this line. Would suggest to run `sphinx` locally to debug this:

```
Warning, treated as error:
/home/vsts/work/1/s/docs/source/user/installing.rst:191:Unexpected indentation.
Makefile:53: recipe for target 'html' failed
make: *** [html] Error 2
```